pages/about/jobs/staff-academic-jobs/index.vue

```diff
@@ -0,0 +1,170 @@
+<script setup>
+// HELPERS
+import _get from 'lodash/get'
+import fixUri from '../utils/fixUri'
+import removeTags from '../utils/removeTags'
+
+// GQL
+import JOB_OPPORTUNITIES_LIST from '../gql/queries/JobOpportunitiesList.gql'
+
+const { $graphql } = useNuxtApp()
+
+const { data } = await useAsyncData('job-opportunities-list', async () => {
+  const data = await $graphql.default.request(JOB_OPPORTUNITIES_LIST)
+
+  return data
+})
+
+const page = ref(_get(data.value, 'entry', {}))
+const allJobs = ref(_get(data.value, 'allJobs', {}))
+
```

Comment:
add preview mode watcher on data and update 
const page = ref(_get(data.value, 'entry', {}))
const allJobs = ref(_get(data.value, 'allJobs', {})) with the new value from craft during preview mode enabled